src/localsocketcontroller.h

```diff
@@ -62,7 +79,11 @@ class LocalSocketController final : public ControllerImpl {
   std::function<void(const QString&)> m_logCallback = nullptr;
 
   QTimer m_initializingTimer;
-  uint32_t m_initializingRetry = 0;
+  uint32_t m_initializingInterval = 0;
+
+  // When a message to the daemon expects an immediate response, these
```

Comment:
Are names like m_responseTimeoutTimers or m_responseTimeouts better? The current name seems to indicate that it is a list of responses.

Reply:
Sure, I will rename it.